### ISSUE TITLE
Error message on top of the Dashboard

### ISSUE THREAD

**Issue by erikbotta** [zabbix/zabbix-community-docker#100]
### Description

Error msg

### Steps to Reproduce

1. Upgrade from 3.2.x to 3.4.4
(-FROM monitoringartist/zabbix-xxl:latest
+FROM monitoringartist/dockbix-xxl:latest)

**Expected behavior:** no error msgs

**Actual behavior:** Undefined index: SERVER_PORT [zabbix.php:21 → require_once() → ZBase->run() → ZBase->processRequest() → CView->getOutput() → include() → local_generateHeader() in app/views/layout.htmlpage.php:57] message on dashboard

**Reproduces how often:** Permanent
### Versions

You can get this information from copy and pasting the output of `docker info`.
Also, please include the OS and what version of the OS and kernel you're running.

### Additional Information

Using rancher 1.6.11 and docker 17.03.1-ce on Ubuntu-server 14.04 LTS

### You Docker knowledge level

advanced

![snimek obrazovky 2017-11-15 v 18 45 03](https://user-images.githubusercontent.com/832402/32851402-26cfb598-ca35-11e7-9d1b-76def032974a.png)

**Comment by jangaraj** [comment]
It's an issue of Zabbix - please report it to https://support.zabbix.com. Problem is a recent change DEV-953:
![image](https://user-images.githubusercontent.com/1182932/32852963-00a4e8e8-ca3a-11e7-8719-d49b83301158.png)

Problem is a condition check:
```
	if ((array_key_exists('https', $_SERVER) && ($_SERVER['https'] == 1 || $_SERVER['https'] === 'on'))
			|| $_SERVER['SERVER_PORT'] == 443) {
```

`$_SERVER['SERVER_PORT']` doesn't exist in this case.

**Comment by nneul** [comment]
I had this as well in #102 - temporary stopgap to at least deal with it until upstream zabbix is corrected - run :3.4.2 instead of :latest - that image was built prior to this upstream change. 

Underlying issue looks to be that zabbix is assuming it's running under apache w/ php that always provides SERVER_PORT, and the code for dockbix-xxl is explicitly NOT setting that value in the nginx config. (It actually has a comment about explicitly disabling that setting.)

**Comment by nneul** [comment]
https://github.com/monitoringartist/dockbix-xxl/blob/2a1fe07dba6bd366bb02f960544d10a7ef8f9ff8/Dockerfile/dockbix-xxl/container-files-base/etc/nginx/fastcgi_params#L25
